### PR TITLE
Pass lang attribute to bulletin

### LINKED ISSUE #3428: Radio bulletin - Incorrect lang attribute on visually hidden text on arabic
**Describe the bug**
The radio bulletin on https://www.bbc.com/arabic has some visually hidden text with a lang attribute stating english as the language of this content, this attribute should not be there as the content is not in english.

**Expected behaviour**
The lang attribute should not be there as the content is not in english.

**Screenshots**
Screen Shot 2020-04-27 at 14 45 56

**Testing notes**

- [ ] This bug fix is expected to need manual testing.

### PR DESCRIPTION
**Pull request by EinsteinNjoroge**
Resolves https://github.com/bbc/psammead/issues/3428

**Overall change:**
_Bump psammead-bulletin and pass lang attribute to bulletin if ctaText is in english._

**Code changes:**

- _Pass `lang` prop to `psammead-bulletin`  if `ctaText` is in english._
- _Bump `psammead-bulletin` to V3.1.0._
- _Update snapshots._

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] I have added labels to this PR for the relevant pod(s) affected by these changes
- [x] I have assigned this PR to the Simorgh project

**Testing:**

- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`)
- [ ] This PR requires manual testing

### CODE REVIEW

**Review by Bopchy — APPROVED** [review]
👍

File: src/app/containers/Bulletin/__snapshots__/index.test.jsx.snap [hunk 1]
```diff
@@ -968,7 +968,7 @@ exports[`Bulletin Container snapshots should render a Radio bulletin correctly 1
         >
           <span
             class="c6"
-            lang="en-GB"
+            lang="ig"
```
Comment (greenc05) [inline src/app/containers/Bulletin/__snapshots__/index.test.jsx.snap]: If this content is in the same language as the page it doesn't need a lang attribute

Comment (DenisHdz) [reply src/app/containers/Bulletin/__snapshots__/index.test.jsx.snap]: If this is the case, I think that we just have to pass the lang attribute as `en-GB` when the translation of the `ctaText` for the specific service is in English, either `Watch` or `Listen`.

```
  const ctaTextIsEnglish = ctaText === 'Watch' || ctaText === 'Listen';
   <Bulletin
      ...
     lang={ctaTextIsEnglish ? 'en-GB' : null}
  />
```


So I guess we will have to remove the default prop `lang: 'en-GB'` in the bulletin component.

**Review by OlgaLyubin — COMMENTED** [review]
Looks good, just need to fix Jenkins cause it's complaining about this linting problem

Screenshot 2020-04-29 at 09 51 27

File: src/app/containers/FrontPageStoryRows/__snapshots__/index.test.jsx.snap [hunk 2]
```diff
@@ -571,7 +571,7 @@ exports[`FrontPageStoryRows Container - snapshots LeadingRow - rtl 1`] = `
             >
               <span
                 class="c8"
-                lang="en-GB"
+                lang="ar"
```
Comment (greenc05) [inline src/app/containers/FrontPageStoryRows/__snapshots__/index.test.jsx.snap]: Is this correct?

File: src/app/containers/Bulletin/__snapshots__/index.test.jsx.snap [hunk 3]
```diff
@@ -1304,7 +1303,7 @@ exports[`Bulletin Container snapshots should render a TV bulletin correctly 1`]
...
```
Comment (greenc05) [inline src/app/containers/Bulletin/__snapshots__/index.test.jsx.snap]: Is this right? The language attribute is set on the document, we only then need to apply this againt if the content has a different language to the docuement

File: src/app/containers/FrontPageStoryRows/__snapshots__/index.test.jsx.snap (hunk 2 above)
Comment (EinsteinNjoroge) [reply src/app/containers/FrontPageStoryRows/__snapshots__/index.test.jsx.snap]: Sorry @greenc05 this is still work in progress

File: src/app/containers/Bulletin/__snapshots__/index.test.jsx.snap (hunk 3 above)
Comment (EinsteinNjoroge) [reply src/app/containers/Bulletin/__snapshots__/index.test.jsx.snap]: https://github.com/bbc/simorgh/pull/6330#discussion_r417211300

**Review by greenc05 — APPROVED** [review]
Snapshots look good thanks @EinsteinNjoroge

**Review by DenisHdz — APPROVED** [review]
Great work 👍

**Comment by PriyaKR** [comment]
LGTM.

**Pull request merged**